### ISSUE TITLE
Resume training bug

### ISSUE THREAD

**Issue by ImmNaruto** [hpcaitech/Open-Sora#863]
Hello, I've encountered an issue with continuing the training. 
Initially, I fine-tuned the model using the Open-Sora-v2 weights and stopped the training after obtaining the step weights. training command use this:
```python
torchrun --nproc_per_node 8 scripts/diffusion/train.py configs/diffusion/train/stage2_i2v.py --dataset.data-path datasets/pexels_45k_necessary.csv --model.from_pretrained ckpts/Open_Sora_v2.safetensors
```
However, when I resumed training using these weights, the loss suddenly surged to a very high value. It took many steps to reduce the loss, but it couldn't return to the original level. resume training command use this:
```python
torchrun --nproc_per_node 8 scripts/diffusion/train.py configs/diffusion/train/stage2_i2v.py --dataset.data-path datasets/pexels_45k_necessary.csv --model.from_pretrained ckpts/Open_Sora_v2.safetensors --load outputs/250521_161822-diffusion_train_i2v/epoch0-global_step1000
```
Do you know what might be causing this issue?
